spec/factories/mentor_memberships.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: mentor_memberships
+#
+#  id         :uuid             not null, primary key
+#  type       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  mentor_id  :uuid             not null
+#  school_id  :uuid             not null
+#
+# Indexes
+#
+#  index_mentor_memberships_on_mentor_id                         (mentor_id)
+#  index_mentor_memberships_on_school_id                         (school_id)
+#  index_mentor_memberships_on_type_and_mentor_id                (type,mentor_id)
+#  index_mentor_memberships_on_type_and_school_id_and_mentor_id  (type,school_id,mentor_id) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (mentor_id => mentors.id)
+#  fk_rails_...  (school_id => schools.id)
+#
+FactoryBot.define do
+  factory :mentor_membership do
+    association :school
+    association :mentor
+
+    trait :claims do
+      type { Claims::MentorMembership }
+    end
+
+    trait :placements do
+      type { Placements::MentorMembership }
+    end
+  end
```

Comment:
Could we add `:claims_mentor_membership` and `:placements_mentor_membership` factories too please?

Most of the other factories are called with the prefix, rather than the trait.

Reply:
0dfcc333083a61294db041c68cc2b7f4627cf2cb